ios/swift/JitsiSDKTest/src/ViewController.swift

```diff
@@ -58,28 +67,37 @@ class ViewController: UIViewController {
             // builder.audioMuted = true;
             // builder.videoMuted = true;
         }
-                
-        // setup view controller
-        let vc = UIViewController()
```

Comment:
Aren't our SDK users mostly using a VC? Wouldn't it make sense to have a JitsiMeetViewController that used the coordinator?

How is Brave integrating this?

Reply:
You can have another view controller and split the functionality between the 2, but is not required. You would have one view controller which only reads the meeting name and one that deals with setting up and presenting the meeting. This is the simplest example. Brave is doing it like this, the BrowserViewController instantiates JitsiMeetView and presents it through the pipViewCoordinator. Not sure about our other users though.

Reply:
So they do have a dedicated VC then. Because the browser is probably a different VC, right?

I'd like to have them separate please, looks like a more useful example for complex apps IMHO.

Reply:
from what I see they don't have a different VC

Reply:
What about WorkM ?

Reply:
we have a custom implementation there, we're not using PiPViewCoordinator

Reply:
Ack, let's have it this way for now then!